CHANGELOG.md

```diff
@@ -3,6 +3,102 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.0.0-alpha.6
+
+_Dec 22, 2023_
+
+We'd like to offer a big thanks to the 6 contributors who made this release possible. Here are some highlights ✨:
+
+- 🌍 Improve Russian (ru-RU) locale on the Data Grid
+- 🐞 Bugfixes
+
+From https://github.com/mui/mui-x/pull/11330
```

Comment:
You can remove this (based on previous changelogs)

Reply:
You sure? I don't seem to find that change in older entries of the changelog?

Reply:
I think Danail's point was to remove the "From ..." lines from the changelog 🙂